pnpm-workspace.yaml

```diff
@@ -126,6 +127,7 @@ catalog:
   js-yaml: ^4.1.0
   jscodeshift: ^0.15.2
   jsdom: ^24.1.3
+  json2csv: 6.0.0-alpha.2
```

Comment:
Why are we using alpha release? And is this better than the more [popular](https://www.oneschema.co/blog/top-5-javascript-csv-parsers) libs like papaparse?